Examples/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
In the current build system, running `make install` installs the examples binaries as well:
https://github.com/seladb/PcapPlusPlus/blob/master/mk/install.sh.template#L15-L20

I think you should add this functionality here also...

Reply:
done